packages/connectivity_plus/connectivity_plus/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Tue Feb 20 19:54:18 GST 2024
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.1-bin.zip
 distributionPath=wrapper/dists
-zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.2-bin.zip
 zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
```

Comment:
This should be OK, but I am not sure if we want to upgrade gradle in this PR, or would be better to do it on a separate PR. Any thoughts @vbuberen

Reply:
Yes, agree here. Even though we plan to move to 8.x in next release I would like it to not be mixed with the topic of this PR.

Reply:
Agreed. I reverted it back.